### PR TITLE
Add modified date range picker

### PR DESCRIPTION
**Pull request by callumdmay**
## Summary

- Remove time input in calendar
- Add buttons for one way or roundtrip
- Add ability to set datepicker trip type
- Add navigation chevrons
- Add UI delay to date

### CODE REVIEW

File: daterangepicker.js [hunk 1]
```diff
@@ -148,6 +139,12 @@
             if (typeof options.locale.monthNames === 'object')
               this.locale.monthNames = options.locale.monthNames.slice();
 
+            if(typeof options.locale.oneWay === 'string')
+              this.locale.oneWay = options.locale.oneWay.slice();
+
+            if(typeof options.locale.roundTrip === 'string')
+              this.locale.roundtrip = options.locale.roundTrip.slice();
```
Comment (nicolashery) [inline daterangepicker.js]: Why call `slice()`?

File: daterangepicker.js [hunk 2]
```diff
@@ -521,6 +519,18 @@
             this.updateMonthsInView();
         },
 
+        setSingleDatePicker: function(is_single_date) {
```
Comment (nicolashery) [inline daterangepicker.js:522]: I don't see this used anywhere?

File: daterangepicker.js [hunk 3]
```diff
@@ -1142,7 +1151,10 @@
 
             $(document).off('.daterangepicker');
             $(window).off('.daterangepicker');
-            this.container.hide();
+            this.container.delay(200).hide({
```
Comment (nicolashery) [inline daterangepicker.js]: What is the reason of the delay? If for something else than pure UI, maybe leave a comment for context?

**Review by nicolashery — APPROVED** [review]
Busbud-specific modifications LGTM!

I think it will be useful to track changes to the original source like this.

🚀

File: daterangepicker.js (hunk 2 above)
Comment (callumdmay) [reply daterangepicker.js:522]: This method is exposed externally, and is used by the `search-form-jquery-client` to change the trip type

**Pull request merged**